enterprise/server/scheduling/scheduler_server/scheduler_server.go

```diff
@@ -1102,7 +1103,7 @@ func (s *SchedulerServer) LeaseTask(stream scpb.Scheduler_LeaseTaskServer) error
 			}
 
 			// Prometheus: observe queue wait time.
-			ageInMillis := (time.Now().UnixNano() / 1e6) - (task.queuedTimestamp.UnixNano() / 1e6)
+			ageInMillis := time.Since(task.queuedTimestamp) / time.Millisecond
```

Comment:
how about:

ageInMillis := time.Since(task.queuedTimestamp).Milliseconds() 

so we're not dividing time.Duration types